usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1489,18 +1497,23 @@
         ),
     },
 }
-
 # IDV Orders are nearly identical to awards but start from the Awards table
 # instead of from UniversalAwardView materialized view so we need to lop off
 # the leading "award__" bit.
 query_paths["idv_orders"] = {
-    "d1": OrderedDict([(k, v[7:] if v.startswith("award__") else v) for k, v in query_paths["award"]["d1"].items()])
+    "d1": OrderedDict(
+        [(k, v[7:] if v.startswith("award__") else v) for k, v in query_paths["award"]["d1"].items() if v is not None]
+    )
 }
 
 # Likewise, IDV Transactions start directly in TransactionFPDS instead of
 # UniversalTransactionView.
 query_paths["idv_transaction_history"] = {
     "d1": OrderedDict(
-        [(k, v[13:] if v.startswith("transaction__") else v) for k, v in query_paths["transaction"]["d1"].items()]
+        [
+            (k, v[13:] if v.startswith("transaction__") else v)
+            for k, v in query_paths["transaction"]["d1"].items()
+            if v is not None
```

Comment:
@emily-brents pointed out that this might introduce risk compared to sentinel value (probably some singleton class). I'm not sure exactly how this will break on the loop unless we start deriving values for the map from functions, but it is possible.

Reply:
I have some ideas on how I can rework this and accomplish the same effect.

Reply:
Total 180 here, but can we save that for a later PR? I want to branch off of this without conflicting so I would rather this branch be shorter-lived.

Reply:
Absolutely. If there are any minor issues I can fix those after we are done with this PR.